tsfresh/examples/robot_execution_failures.py

```diff
@@ -102,10 +102,7 @@ def load_robot_execution_failures():
             if line[0] not in ['\t', '\n']:
                 cur_id += 1
                 time = 0
-                if line.strip() == 'normal':
-                    id_to_target[cur_id] = 0
-                else:
-                    id_to_target[cur_id] = 1
+                id_to_target[cur_id] = line.strip()
```

Comment:
Yes, here we return the class label. Before we only returned failure/no failure. As we are now capable of selection for multiclassification, the robot execution failure dataset can serve as an example for that.